backend/canisters/community/api/can.did

```diff
@@ -132,7 +132,6 @@ type SelectedChannelInitialResponse = variant {
         blocked_users : vec UserId;
         invited_users : vec UserId;
         pinned_messages : vec MessageIndex;
-        rules : Rules;
```

Comment:
Has this already been removed from the FE?

Reply:
no - needs website to go next